systemtest/src/test/java/io/strimzi/systemtest/security/PodSecurityProfilesST.java

```diff
@@ -60,169 +68,158 @@ public class PodSecurityProfilesST extends AbstractST {
+     *     - All containers and Pods have expected properties
+     *  6. - Verify KafkaConnect and KafkaConnector are working by checking presence of Data in file targeted by FileSink KafkaConnector
+     *     - Data are present here
+     *  7. - Verify Kafka and MirrorMakers by Deploy kafka Consumers in respective target Kafka Clusters targeting KafkaTopics created by mirroring
+     *     - Data are present here
+     *  8. - Deploy Kafka consumer with invalid configuration regarding, i.e., without pod security profile
+     *     - Consumer is unable to communicate correctly with Kafka
+     *
+     * @usecase
+     *  - security-profiles
+     */
     @Tag(ACCEPTANCE)
     @ParallelNamespaceTest
-    // Pod Security profiles works from 1.23 Kubernetes version or OCP 4.11
     @RequiredMinKubeOrOcpBasedKubeVersion(kubeVersion = 1.23, ocpBasedKubeVersion = 1.24)
-    void testKafkaWithRestrictedSecurityProfile(ExtensionContext extensionContext) {
-        final TestStorage testStorage = new TestStorage(extensionContext);
-        addRestrictedPodSecurityProfileToNamespace(testStorage.getNamespaceName());
-
-        // if Kafka Pods deploys it means that it works...
-        resourceManager.createResourceWithWait(extensionContext, KafkaTemplates.kafkaPersistent(testStorage.getClusterName(), 3).build());
-
-        // 1. check the generated structure of SecurityContext of Kafka Pods
-        // verifies that (i.) Pods and (ii.) Containers has proper generated SC
-        verifyPodAndContainerSecurityContext(PodUtils.getKafkaClusterPods(testStorage));
-
-        // 2. check that Kafka cluster is usable and everything is working
-        verifyStabilityOfKafkaCluster(extensionContext, testStorage);
-    }
+    void testOperandsWithRestrictedSecurityProfile(ExtensionContext extensionContext) {
 
-    @ParallelNamespaceTest
-    @RequiredMinKubeOrOcpBasedKubeVersion(kubeVersion = 1.23, ocpBasedKubeVersion = 1.24)
-    void testKafkaWithKafkaBridgeRestrictedSecurityProfile(ExtensionContext extensionContext) {
         final TestStorage testStorage = new TestStorage(extensionContext);
-        addRestrictedPodSecurityProfileToNamespace(testStorage.getNamespaceName());
 
-        // if Kafka and KafkaBridge Pods deploys it means that it works...
-        resourceManager.createResourceWithWait(extensionContext, KafkaTemplates.kafkaPersistent(testStorage.getClusterName(), 3).build());
-        resourceManager.createResourceWithWait(extensionContext, KafkaBridgeTemplates.kafkaBridge(testStorage.getClusterName(),
-            KafkaResources.plainBootstrapAddress(testStorage.getClusterName()), 1).build());
+        final String mm1TargetClusterName = testStorage.getTargetClusterName() + "-mm1";
+        final String mm2TargetClusterName = testStorage.getTargetClusterName() + "-mm2";
+        final String mm2SourceMirroredTopicName = testStorage.getClusterName() + "." + testStorage.getTopicName();
+        final int messageCount = 100;
```

Comment:
Isn't this count in `Constants`? And also in `testStorage`

Reply:
these are the names of actual Kafka Clusters not mirror makers as such, 
regarding message count i wanted it to be constant 100 hundred regardless of further possible changes in testStorage as i also use the same number for later connector verification.